app/routes/users/components/UserSettings.tsx

```diff
@@ -130,8 +138,10 @@ const UserSettings = (props: Props) => {
                 component={RadioButton.Field}
               />
             </RadioButtonGroup>
+
             <Field
               label="Matallergier/preferanser"
+              parse={(value) => value}
```

Comment:
Could you explain why this is necessary? I don't doubt that it is, but does seem a bit weird.

Reply:
I see, this is good to know!